packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -933,6 +959,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
     }
 
+    private formMetadata(): IContainerRuntimeMetadata {
+        return {
+            summaryFormatVersion: 1,
+            disableIsolatedChannels: this.disableIsolatedChannels || undefined,
```

Comment:
seems a bit strange to specify the feature in the metadata, shouldn't the version capture that?

Reply:
We will have at least 2 features encoded here at the same time. I don't want to limit ourselves here to having both or neither.